Description: source typo
 Correct spelling error as reported by lintian in some binraries;
 meant to silence lintian and eventually to be submitted to the
 upstream maintainer.
Origin: debian
Forwarded: email
Comment: spelling-error-in-binary
Author: Jerome Benoit <calculus@rezozer.net>
Last-Update: 2016-08-23

--- a/genbg.c
+++ b/genbg.c
@@ -23,7 +23,7 @@
   -L    : there is no vertex in the first class whose removal leaves\n\
           the vertices in the second class unreachable from each other\n\
   -Z#   : two vertices in the second class may have at most # common nbrs\n\
-  -A    : no vertex in the second class has a neighbourhood whith is a\n\
+  -A    : no vertex in the second class has a neighbourhood which is a\n\
           subset of another vertex in the second class\n\
   -D#   : specify an upper bound for the maximum degree\n\
           Example: -D6.  You can also give separate maxima for the\n\
